app/src/sharedTest/java/org/oppia/android/app/profile/AddProfileActivityTest.kt

```diff
@@ -130,6 +131,19 @@ class AddProfileActivityTest {
     ApplicationProvider.getApplicationContext<TestApplication>().inject(this)
   }
 
+  @Test
+  fun testAddProfileActivity_hasCorrectActivityLabel() {
```

Comment:
Suggest using `ActivityTestRule` similar to https://github.com/oppia/oppia-android/blob/9b7d57ecd1f00ed630c1b95d56e4cdfc4c84e8c0/app/src/sharedTest/java/org/oppia/android/app/completedstorylist/CompletedStoryListActivityTest.kt#L144

Reply:
@rt4914 To launch `AddProfileActivity` using `ActivityTestRule`, we need an intent.
`fun createAddProfileActivityIntent(context: Context, colorRgb: Int): Intent` requires a `colorRgb` value.
Other callers of this function call it using:
```
activity.startActivity(
  AddProfileActivity.createAddProfileActivityIntent(
    context, activity.intent.getIntExtra(ADMIN_AUTH_COLOR_RGB_EXTRA_KEY, -10710042)
  )
)
```
Should I pass the default value in the test?

Reply:
This reference can help on how to create intent: https://github.com/oppia/oppia-android/blob/9b7d57ecd1f00ed630c1b95d56e4cdfc4c84e8c0/app/src/sharedTest/java/org/oppia/android/app/completedstorylist/CompletedStoryListActivityTest.kt#L455

Reply:
@rt4914 Yeah, I saw that. But `createAddProfileActivityIntent()` requires a colorRgb value to create an intent. What should I pass there?

Reply:
The value is mentioned in the above code that you shared. `-10710042`

Reply:
Done